crates/sim2h/src/cache.rs

```diff
@@ -208,10 +208,10 @@ impl AspectList {
     }
 
     pub fn add(&mut self, entry_address: EntryHash, aspect_address: AspectHash) {
-        self.0
-            .entry(entry_address)
-            .or_insert_with(Vec::new)
-            .push(aspect_address);
+        let list = self.0.entry(entry_address).or_insert_with(Vec::new);
```

Comment:
I still think we should just change this type to a hashset.

Reply:
I would like to very much, but it's not just a trivial change because it flows through to a lot of things..